scripts/kokkos_cuda_openmp_cmake

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+##---------------------------------------------------------------------------##
+## CONFIGURE DTK
+##---------------------------------------------------------------------------##
+
+# This script will build DTK with Kokkos support for Serial, OpenMP, and CUDA
+# node types.
+
+# NOTE: The CUDA hardware architecture should be set in CMAKE_CXX_FLAGS. The
+#       current flag setting is for compute capability 3.0. Change for your
+#       architecture.
+
+# NOTE: Before running this you must source set_kokkos_env.sh to set the NVCC
+#       wrapper as the MPI C++ compiler implementation and your C++ compiler
+#       as the NVCC compiler implementation. It will also set CUDA calls as 
+#       blocking.
+#
+#       https://trilinos.org/about/documentation/building-with-cuda-support/
+
+rm -rf CMakeCache.txt
+rm -rf CMakeFiles
+
+##---------------------------------------------------------------------------##
+
+cmake \
+    -D CMAKE_INSTALL_PREFIX:PATH=/home/uy7/install/DataTransferKit/kokkos \
+    -D CMAKE_BUILD_TYPE:STRING=DEBUG \
+    -D CMAKE_VERBOSE_MAKEFILE:BOOL=OFF \
+    -D BUILD_SHARED_LIBS:BOOL=OFF \
+    -D CMAKE_CXX_COMPILER=/usr/bin/mpicxx \
+    -D CMAKE_C_COMPILER=/usr/bin/mpicc \
+    -D CMAKE_Fortran_COMPILER=/usr/bin/mpif90 \
+    -D CMAKE_CXX_FLAGS="-g -arch=sm_30 -lineinfo \
+-Xcudafe --diag_suppress=conversion_function_not_usable \
+-Xcudafe --diag_suppress=cc_clobber_ignored \
+-Xcudafe --diag_suppress=code_is_unreachable" \
+    -D TPL_ENABLE_MPI:BOOL=ON \
+    -D TPL_ENABLE_Boost:BOOL=ON \
+    -D Boost_LIBRARY_DIRS:PATH=/home/uy7/install/boost_1_59_0/lib \
```

Comment:
Instead of hardcoding the path that is only valid for your machine can you do sth like in [docker_cmake](https://github.com/ORNL-CEES/DataTransferKit/blob/master/scripts/docker_cmake)